modules/ROOT/pages/logs.adoc

```diff
@@ -268,12 +267,12 @@ image::log-filter-add-field.png[To Add a Filter Column to the Logs]
 
 The resulting `log-level` column looks like this in the logs:
 
-image::log-filter-column.png[Filter Column in the Logs]
+image::log-filter-column.png["Filter column in the logs"]
 
 To remove the filter column, click the column icon for the filter field,
 for example:
 
-image::log-filter-remove-field.png[To Remove a Filter Column from the Logs]
+image::log-filter-remove-field.png["Removing a filter column from the logs"]
```

Comment:
Instead of these gerund constructions, maybe make it more descriptive - Filter field column icon

Reply:
```suggestion
image::log-filter-remove-field.png["Column icon for removing the column from the logs"]
```